src/tpm2tss-genkey.c

```diff
@@ -151,6 +158,20 @@ parse_opts(int argc, char **argv)
                 exit(1);
             }
             break;
+        case 'm':
+            if (strcasecmp(optarg, "cfb") == 0) {
+                opt.alg = TPM2_ALG_CFB;
```

Comment:
Shouldn't this all be opt.sym_mode ?

Reply:
aah yes sorry I have seen this mistake I have already fixed it in my other branch. Either I merge my branch in master (but CI not currently passing for openssl 1.0.2), either I correct it directly in master

Reply:
also there are some other errors when managing sym mode